lib/sneakers/workergroup.rb

```diff
@@ -52,5 +54,13 @@ def stop
       @stop_flag.set!
     end
 
+    def create_bunny_connection
+      Bunny.new(Sneakers::CONFIG[:amqp], :vhost => Sneakers::CONFIG[:vhost], :heartbeat => Sneakers::CONFIG[:heartbeat], :logger => Sneakers::logger)
+    end
+
+    def create_connection_or_nil
```

Comment:
I'm afraid it's not obvious what the effects of passing in a `Bunny::Session` instance vs. a `nil` are. Can you please add a comment explaining that a non-`nil` connection will be used as is, and a `nil` will ensure workers open their own?